### ISSUE TITLE
Alarm sounds while phone is muted

### ISSUE THREAD

**Issue by socialize-IT** [Futsch1/medTimer#319]
**Is your feature request related to a problem? Please describe.**
As I mute my phone quite often it's hard to get all the reminders for my pills.

**Describe the solution you'd like**
I would like to have not only the notification, but also an alarm for my high-priority meds. That means I can hear an alarm sound even if I muted my phone or put it in vibration mode. At least on samsung phones my wakeup-alarms are ringing even if the phone is muted. I would also like that for my medication notifications..

**Comment by Futsch1** [comment]
For some Android devices, the notification settings provide the option to allow the notification during Do not Disturb mode:

<img src='https://github.com/user-attachments/assets/68399da1-1ee8-4092-991a-0664023a79da' width=300/>


For Samsung phones, I found it is possible to also explicitly exclude apps from this mode:

<img src='https://github.com/user-attachments/assets/032e1095-2641-417d-a04e-a63c5899d2a9' width=300/>

Have you tried this?

**Comment by socialize-IT** [comment]
Yes, I know this, but this doesn't meet my usecase. I have my phone in vibration mode like the whole time to be available for calls. But sometimes I put my phone on a table or anything. Then I can call back later but need the alarm for my meds..

**Comment by timtoo** [comment]
An actual alarm would be good for me too. I've put this app on my mom's phone, and she misses notifications all the time. I've put the app on high priority notifications, and picked the most intrusive notification sound, and default action to snooze, and maximum number of snoozes, and configured notifications to ignore do-not-disturb (quite a lot of config changes), but still it's not enough. I need an alarm that will be loud for half an hour until it is attended to! Anyhow, my work around at the moment is setting a alarm in the android clock, and keeping this nice app as well for tracking.

**Comment by Futsch1** [comment]
> Ok, I see. I will investigate some open source alarm apps and see what they do to make this work. But I suppose this is a major change, so I cannot promise an arrival date.

Did some research, it is actually much easier than I thought to bypass do not disturb. So @socialize-IT, I will be able to solve your use case in the release after the upcoming one.

@timtoo You actually aim for something more than just a sound during do not disturb if I understand correctly. So I will create a new issue once this is resolved that aims for a "real" alarm where the sound is repeated for a longer period of time.

**Comment by socialize-IT** [comment]
@Futsch1 : That sounds nice, but important for me is not to bypass dnd-mode! As I often mute my phone or put it in vibration mode. Otherwise I could use the exceptions for dnd-mode.

![Image](https://github.com/user-attachments/assets/fa137d45-d19f-47ff-9366-33165bc78fad)

**Comment by socialize-IT** [comment]
//Accidentally closed - sorry

**Comment by Futsch1** [comment]
> [@Futsch1](https://github.com/Futsch1) : That sounds nice, but important for me is not to bypass dnd-mode! As I often mute my phone or put it in vibration mode. Otherwise I could use the exceptions for dnd-mode.
> 
> ![Image](https://github.com/user-attachments/assets/fa137d45-d19f-47ff-9366-33165bc78fad)

I would actually make sure that this does not only bypass dnd mode, but also vibration only (I suppose this might work, but I'll try it).
